src/wp-includes/class-wp-query.php

```diff
@@ -4755,6 +4722,55 @@ public function generate_postdata( $post ) {
 
 		return $elements;
 	}
+
+	/**
+	 * Generate cache key.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param array $cache_args
+	 * @param string $fields
+	 *
+	 * @return string
+	 */
+	protected function generate_cache_key( array $cache_args, $request ) {
```

Comment:
Is it possible to rename `$request` to `$sql_string_for_cache_key` or something?

The use of `$request` confused me with `$this->request` on first reading and I think it would be a problem if the misunderstanding later resulted in a (not) bug fix updating the request property.

Reply:
+1. Better to have a nice, self-documenting var name :)

Reply:
Changed to $sql, it is much cleaner.